src/twisted/web/test/test_web.py

```diff
@@ -777,6 +777,25 @@ def test_encoding(self):
                           zlib.decompress(body, 16 + zlib.MAX_WBITS))
 
 
+    def test_whitespaceInAcceptEncoding(self):
+        """
+        If the client request passes a I{Accept-Encoding} header which mentions
+        gzip, with whitespace inbetween the encoding name and the commas,
+        L{server._GzipEncoder} automatically compresses the data.
+        """
+        request = server.Request(self.channel, False)
+        request.gotLength(0)
+        request.requestHeaders.setRawHeaders(b"Accept-Encoding",
+                                             [b"deflate, gzip"])
```

Comment:
Since this is a test of `GzipEncoderFactory` I don't think that this is particularly a problem. (I'd also wager we won't ever add support for deflate, since it's basically supported by a subset of the clients that gzip is, and historically clients are buggy. Brotli&mdash;`br`&mdash;would be a more useful addition.)